.github/workflows/release_pr.yml

```
@@ -0,0 +1,50 @@
+name: Create PR for a release
+on:
+  workflow_dispatch:
+env:
+  GIT_USER_NAME: amplify-android-dev+ghops
+  GIT_USER_EMAIL: amplify-android-dev+ghops@amazon.com
+  BASE_BRANCH: main
+jobs:
+  create_pr_for_next_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Update git
+      run: |
+        sudo add-apt-repository -y ppa:git-core/ppa
+        sudo apt-get update
+        sudo apt-get install git -y
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ env.BASE_BRANCH }}
+        fetch-depth: 0
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: Install dependencies
+      run: |
+        cd scripts
+        gem install bundler
+        bundle install
+    - name: Configure git options
+      run: |
+        cd scripts
+        bundle exec fastlane android configure_git_options git_user_email:$GIT_USER_EMAIL git_user_name:$GIT_USER_NAME
+    - name: Create/checkout a branch for the release
+      run: |
+        branch_name=release
```

Naming this branch `release` doesn't seem appropriate, since no release is actually happening - it's just bumping the version.

```suggestion
        branch_name=bump_version
```